python/ray/train/base_trainer.py

```diff
@@ -571,7 +571,7 @@ def fit(self) -> Result:
             A Result object containing the training result.
 
         Raises:
-            TrainingFailedError: If any failures during the execution of
+            TrainingFailedError\: If any failures during the execution of
```

Comment:
Why are we adding a `\` here?

Reply:
Because it's not valid RST syntax without it.

Reply:
Actually, I take it back: we're using `napoleon` to allow Sphinx to ingest google-style docstrings, so this should be okay. The real culprit is twofold:

1. The line underneath needs to be indented if it's part of the same explanation
2. The exception needs to be fully qualified

Making changes now...

Reply:
Resolved!